VultisigApp/VultisigApp/Services/BlockChainService.swift

```diff
@@ -347,8 +364,24 @@ private extension BlockChainService {
             guard let sequence = UInt64(account?.sequence ?? "0") else {
                 throw Errors.failToGetSequenceNo
             }
-            return .Cosmos(accountNumber: accountNumber, sequence: sequence, gas: 7500, transactionType: transactionType.rawValue, ibcDenomTrace: nil)
             
+            var ibcDenomTrace: CosmosIbcDenomTraceDenomTrace? = nil
```

Comment:
these logic are the same as line 312-326, probably add a function , so you don't need to copy it